### ISSUE TITLE
The simplified stack traces for failing cases should start from the test method

### ISSUE THREAD

**Issue by testforstephen** [microsoft/vscode-java-test#1415]
![image](https://user-images.githubusercontent.com/14052197/165247941-a641d9f8-ae1a-47c0-9e62-a5bf10b442b8.png)

The highlighting two stacks don't provide any valid info for diagnose, they can be hidden as well.

**Comment by jdneo** [comment]
Agree, the last two lines does not provide much help in this case.

Meanwhile, in IntelliJ and Eclipse, they both have the same simplified results in this case. Might be some reason that we don't know so far. I would like to wait some more feedbacks from users and decide the next step.

![image](https://user-images.githubusercontent.com/6193897/165279106-7e8fef4a-b852-43bf-892c-45cb22286df3.png)


![image](https://user-images.githubusercontent.com/6193897/165279450-f681044f-586c-44d4-994f-c895cb9d751a.png)
